account_banking_pain_base/migrations/11.0.2.0.0/post-migrate.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    """Update database from previous versions, after updating module."""
+
+    if not version:
+        return
+    cr = env.cr
+
+    # update charge_bearer of account_payment_lines
+    plines = env['account.payment.line'].search([])
+    for pline in plines:
+        pline.compute_charge_bearer()
+
+    # update charge_bearer for all bank_payment_line
+    cr.execute(
+        "UPDATE bank_payment_line as l "
+        "SET charge_bearer = ("
+        "SELECT charge_bearer FROM account_payment_order as o "
+        "WHERE o.id = l.order_id)")
+    # correct charge_bearer for sepa bank_payment_line (must be 'SLEV')
+    blines = env['bank.payment.line'].search([])
+    for bline in blines:
+        if bline.sepa:
+            bline.charge_bearer = 'SLEV'
```

Comment:
I would do this in SQL too: `UPDATE bank_payment_line SET charge_bearer = 'SLEV' where sepa`?
That said, if there was incorrect data, it's maybe not the role of this PR to fix it?

Reply:
`sepa` is a computed field so it's not present in database.

Reply:
In fact charge_bearer is a new field in bank.payment.line and its default value is 'SLEV' so I guess this can be removed.